wpa_supplicant/ctrl_iface_zephyr.c

```diff
@@ -175,12 +248,32 @@ wpa_supplicant_global_ctrl_iface_init(struct wpa_global *global)
 	priv->global = global;
 	memset(priv->sock_pair, -1, sizeof(priv->sock_pair));
 
-	ret = socketpair(AF_UNIX, SOCK_STREAM, 0, priv->sock_pair);
-	if (ret != 0) {
-		wpa_printf(MSG_ERROR, "socket(PF_INET): %s", strerror(errno));
+	ret = zvfs_eventfd(0, 0);
```

Comment:
Doesn't this tie to ZVFS? Shouldn't we be using implementation neutral APIs? Also, I don't see in the Kconfig.

```
	select ZVFS
	select ZVFS_EVENTFD
```

Reply:
Yes, we can add the selects, they came automatically by some other dependency but should be mentioned here too.